src/components/CustomerQuota/CustomerQuotaScreen.tsx

```diff
@@ -177,10 +177,28 @@ const CanBuyResult: FunctionComponent<{
       <CustomerCard nric={nric}>
         <View style={styles.resultWrapper}>
           {Object.entries(cart)
-            .sort()
+            .sort((itemOne, itemTwo) => {
```

Comment:
Added sorting logic here. Previously we were sorting by ascending ascii character. 

Currently, the backend will return the `name` field that will be prepended with a number (like `1. XXX`). This function will extract out the number and order it accordingly

Reply:
I'm wondering if we want to change the `Product` interface instead, and add the order there. This will make it easier to sort. We're likely to need it when sorting the purchased products too.

Reply:
Ohhh, I see I see! Okk will make the changes!

Reply:
Made the changes!

Reply:
👍